pkg/cmd/server/admin/create_signercert.go

```diff
@@ -99,9 +96,9 @@ func (o CreateSignerCertOptions) CreateSignerCert() (*crypto.CA, error) {
 		ca, written, err = crypto.EnsureCA(o.CertFile, o.KeyFile, o.SerialFile, o.Name)
 	}
 	if written {
-		fmt.Fprintf(o.Output.Get(), "Generated new CA for %s: cert in %s and key in %s\n", o.Name, o.CertFile, o.KeyFile)
+		glog.V(3).Infof("Generated new CA for %s: cert in %s and key in %s\n", o.Name, o.CertFile, o.KeyFile)
```

Comment:
> We have log levels for this.

I'd tend to disagree.  When you explicitly running create-signer-cert, you clearly want this message.  When you're delegating to this method because you ran an `openshift start server`, the relative importance of these messages changes.  That can be handled by not having a library type method perform output, but that seems heavy in this case.  It could also be handled by a filtering chain on top of an io.Writer which seems far more reasonable in this case.